libr/hash/luhn.c

```diff
@@ -0,0 +1,22 @@
+/* Copyright (C) radare2 - 2007-2016 - pancake */
```

Comment:
this copyright holder is not correct. i didnt wrote this code in 2007 :P put your nick/name and just 2016 in there